src/msix/AppxManifestObject.cpp

```diff
@@ -368,6 +368,30 @@ namespace MSIX {
         return static_cast<HRESULT>(Error::NotImplemented);
     }
 
+    // IAppxManifestReader4
+    HRESULT STDMETHODCALLTYPE AppxManifestObject::GetOptionalPackageInfo(IAppxManifestOptionalPackageInfo **optionalPackageInfo) noexcept try

```

Comment:
> AppxManifestObject::GetOptionalPackageInfo [](start = 29, length = 43)

So for tests... luckily (for you) the test got refactored in the packaging branch, so I wouldn't recommend it adding it now unless you want to deal with merge conflicts later. 

Context: Manifest test used to be in api_package.cpp but, in packaging, they are in api_manifestreader.cpp. The new ones take an actual xml file, as opposed to opening a package every time.

We are targeting to pull packaging into master hopefully early next week. Then you can add tests :)

Reply:
Ok sounds good. I will add the tests next week.